src/index.ts

```diff
@@ -381,7 +381,9 @@ export class Upload extends Pumpify {
         this.destroy(new Error('Upload failed'));
         return;
       }
-
+      if (resp && resp.data) {
+        resp.data.size = Number(resp.data.size);
```

Comment:
Generally I don't go too crazy here, but isn't this technically a breaking change?  @bcoe trust whatever you decide :)

Reply:
I think I'm okay with this as a patch, because anyone consuming the library is most likely calling:

`Number(resp.data.size)`

Which will continue to work with `size` already being converted to a number `Number(10) === 10`.

At the same time, I think this makes upstream code safer for folks who forget to convert the value.